### ISSUE TITLE
Feature Request - Part attribute unloader

### ISSUE THREAD

**Issue by KatherineLAnderson** [ArctosDB/arctos#8011]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Is your feature request related to a problem? Please describe.**
Context: I have entered specimen locations (i.e., the drawer or shelf the specimen is in) in the "location" part attribute. Sometimes we move specimen parts (sometimes entire drawers) around and need to modify that part attribute in bulk. I'm looking for a way to do this without having to manually search each record and edit the part attribute.

**Describe what you're trying to accomplish** 
A way to bulk unload part attributes 

**Describe the solution you'd like**
Part attribute unloader similar to the locality attribute and record attribute unloaders that already exist

**Describe alternatives you've considered**
Manually editing these would be very time consuming.

**Additional context**
Useful for other collections, minimally UWBM:PB and UWBM:PR (@WaigePilson )

**Priority**
High--needed for day-to-day operations

**Comment by campmlc** [comment]
@KatherineLAnderson Wouldn't it be preferable to add a new part attribute for the new location? That way you would track the history of where the part has been, which could be relevant for enviromental monitoring and documentation, or in case something gets misplaced. 
Note that this capability is in inherent in object tracking, which does all that very well - both tracking current location and location history. And it is easy to move the contents of an entire shelf or rack to a new position without having to edit each one.

**Comment by KatherineLAnderson** [comment]
> @KatherineLAnderson Wouldn't it be preferable to add a new part attribute for the new location? That way you would track the history of where the part has been, which could be relevant for environmental monitoring and documentation, or in case something gets misplaced. 

True, but then we'd still need to modify the part attribute with the previous location to note it was moved (or add a determined date that would be before the determined date of the new location).

> Note that this capability is in inherent in object tracking, which does all that very well - both tracking current location and location history. And it is easy to move the contents of an entire shelf or rack to a new position without having to edit each one.

Yes, in the long run we will be using object tracking which I agree is the best practice way to handle this. This system using part attributes is a stopgap solution between getting the data to a level where we are able to function using Arctos, and implementation of object tracking.

**Comment by WaigePilson** [comment]
I think this feature would be extremely helpful for a number of situations! In general, bulk editing is very helpful for my current workflows and would be extremely helpful once I am fully migrated into Arctos.

I agree with Katie that while object tracking would be useful in this specific instance, 1) we (UWBM) will not be able to implement object tracking for at least a few months past when we are operating in Arctos and 2) in the meantime, I would not like to gum up my specimen records with a lot of minute changes in the location part attribute (move from one drawer to another in the exact same cabinet, sometimes that happens twice in a week!)

Also, bigger picture I think that having a part attribute unloader would be extremely helpful for plenty of other instances. I'm currently bulk importing ~70k legacy specimen records. If/when I make mistakes, it would be great to be able to bulk unload and re-load them.

**Comment by campmlc** [comment]
I understand. I agree with this and think we should have unloaders for all tools to be consistent.

**Comment by KatherineLAnderson** [comment]
Hi @dustymc - what's the status on this issue? thank you!
